packages/mds-types/index.ts

```diff
@@ -149,7 +149,18 @@ export type UUID = string
 export type Timestamp = number
 export type Stringify<T> = { [P in keyof T]: string }
 export type Nullable<T> = T | null
-export type NullableProperties<T extends {}> = { [P in keyof T]: Nullable<T[P]> }
+
+export type DeepNullable<T> = {
```

Comment:
ok i totally cannot read this

Reply:
Conditional mapped type. Read it just like a js/ts ternary expression. In this case, it makes all object properties nullable.

Reply:
I'm guessing this is something like, an object where any arbitrarily nested property can be null, but might be worth a comment if people are getting confused.

Reply:
Some helpy docs https://www.typescriptlang.org/docs/handbook/advanced-types.html#conditional-types

Reply:
I am going to change this back to just `NullableProperties` ... we only need the shallow nullable and it will be much less confusing.